### PR TITLE
adds docu about how to acknowledge a sys.node_checks entry

### PR DESCRIPTION
**Pull request by seut**
Doctests will not pass until functionality is implemented.

### CODE REVIEW

File: blackbox/docs/sql/system.txt [hunk 1]
```diff
@@ -594,6 +595,42 @@ Example query::
...
+what we want on a healthy well running cluster).
+
+In order to make e.g. the Admin-UI accept a failing check (so the
+checks label goes green again), we must acknowledge this check by
+updating it's `acknowledged` flag::
+
+  cr> update sys.node_checks set acknowledged = true where id = 1;
+  UPDATE 1 row affected (... sec)
+
+Of course we can also acknowledge every check on every node (do this
```
Comment (mfussenegger) [inline blackbox/docs/sql/system.txt]: I'd remove the second example

**Comment by mfussenegger** [comment]
lgtm

File: blackbox/docs/sql/system.txt [hunk 2]
```diff
@@ -594,6 +595,42 @@ Example query::
...
+.. _sys-node-checks-ack:
+
+Acknowledge failed checks
+-------------------------
+
+It is possible to acknowledge every check by updating the
+`acknowledged` column. By doing this, specially Crate's built-in
+Admin-UI won't complain anymore about failing checks.
+
+Imagine we've added a new node to our cluster, but as e.g. the
```
Comment (aelesbao) [inline blackbox/docs/sql/system.txt]: change `, but as e.g. the` to `, e.g.,`

File: blackbox/docs/sql/system.txt [hunk 3]
```diff
@@ -594,6 +595,42 @@ Example query::
...
+`gateway.expected_nodes` column can only be set via config-file or
+command-line argument, the check for this setting will not pass on the
+already running nodes until the config-file or command-line argument
+on these nodes is updated and the nodes are restarted (which is not
+what we want on a healthy well running cluster).
+
+In order to make e.g. the Admin-UI accept a failing check (so the
```
Comment (aelesbao) [inline blackbox/docs/sql/system.txt]: I think that `e.g.` here is not necessary

**Comment by aelesbao** [comment]
👍

**Pull request merged**